README.md

```diff
@@ -1 +1,177 @@
-# hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+# 📗 Table of Contents
+
+- [� Table of Contents](#-table-of-contents)
```

Comment:
- [  ] As stated by the previous reviewer, the table of content inside the table of content links is redundant and not needed as users won't need to click on this link when they are already on the table of content. I would suggest you remove it from here as well.
- Just delete this line `10`, since line `8` is the main header 👍